src/game/boe.combat.cpp

```diff
@@ -4030,13 +4021,16 @@ bool monst_cast_priest(cCreature *caster,short targ) {
 	return acted;
 }
 
-void damage_target(short target,short dam,eDamageType type,short sound_type) {
-	if(target == 6) return;
+short damage_target(short target,short dam,eDamageType type,short sound_type, bool do_print, short who_hit, eRace race) {
+	if(target == 6) return 0;
```

Comment:
Come to think of it, I guess this should exclude everything from 6 up to (but not including) 100…